addon/-private/properties/alias.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+
+import {
+  getProperty,
+  objectHasProperty
+} from '../helpers';
+
+/**
+ * Returns the value of some other property on the PageObject.
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continue: alias('continueButton.click')
+ * });
+ *
+ * // calls `page.continueButton.click`
+ * page.continue();
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continueText: alias('continueButton.text')
+ * });
+ *
+ * // checks value of `page.continueButton.text`
+ * assert.equal(page.continueText, 'Go ahead!');
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   clickContinueButton: clickable('button.continue'),
+ *   continue: alias('clickContinueButton')
+ * });
+ *
+ * // calls `clickContinueButton`
+ * page.continue();
+ *
+ * @public
+ *
+ * @param {string} pathToProp - dot-separated path to PageObject property
```

Comment:
There are some differences between `Ember.computed.alias` and `PageObject.alias` that IMHO we should highlight in the documentation so people doesn't get frustrated by things not working as they expect. One I can think of is how we treat collections, with Ember's alias computed property you can do things like:

```js
import Ember from 'ember';

export default Ember.Component.extend({
  posts: Ember.computed(function() {
    return ['hello', 'world', 'foo', 'bar'];
  }),

  firstPost: Ember.computed.alias('posts.firstObject')
});
```

We aren't supporting these _operators_ (`firstObject`, `lastObject`) and I think having a note in the docs mentioning that could save us from some bug reports.

What do you think?